COCOAPODS.md

```diff
@@ -1,150 +1,381 @@
-
-kotlin {
-    cocoapods {
-        // Configure fields required by CocoaPods.
-        summary = "Some description for a Kotlin/Native module"
-        homepage = "Link to a Kotlin/Native module homepage"
-
-        // The name of the produced framework can be changed.
-        // The name of the Gradle project is used here by default.
-        frameworkName = "my_framework"
+You can add dependencies between a Kotlin project and Pod libraries [stored in the CocoaPods repository](#add-a-dependency-on-a-pod-library-from-the-cocoapods-repository) 
+and [stored locally](#add-a-dependency-on-a-pod-library-stored-locally).
+
+[Complete initial configuration](#install-the-cocoapods-dependency-manager-and-plugin), and when you add a new dependency, 
+just re-import the project in IntelliJ IDEA. The new dependency will be added automatically. No additional 
+steps are required.
+
+### Add a dependency on a Pod library from the CocoaPods repository
+
+1. Add dependencies on a Pod library that you want to use from the CocoaPods repository with `pod()`  to `build.gradle.kts` 
+(`build.gradle`) of your project.  
+    > You can also add dependencies on subspecs.
+    {:.note}                                                                                                                                                              >
+
+    <div class="sample" markdown="1" theme="idea" data-highlight-only>
+    
+    ```kotlin
+    kotlin {
+        ios()
+    
+        cocoapods {
+            summary = "CocoaPods test library"
+            homepage = "https://github.com/JetBrains/kotlin"
+            pod("AFNetworking", "~> 4.0.0")
+            
+            pod("SDWebImage/MapKit")
+        }
     }
-}
+    ```
+    
+    </div>
+
+2. Re-import the project.
+
+To use these dependencies from Kotlin code, import packages `cocoapods.<library-name>`.
+
+<div class="sample" markdown="1" theme="idea" data-highlight-only>
+
+```kotlin
+import cocoapods.AFNetworking.*
+import cocoapods.SDWebImage.*
 ```
 
 </div>
 
-The following podspec fields are required by CocoaPods:
-  - `version`
-  - `summary`
-  - `homepage`
+You can find a sample project [here](https://github.com/Kotlin/kotlin-with-cocoapods-sample).
 
-A version of the Gradle project is used as a value for the `version` field.
-Fields`summary` and `homepage` can be configured using the `cocoapods` code block.
+### Add a dependency on a Pod library stored locally
 
-This podspec file can be referenced from a [Podfile](https://guides.cocoapods.org/using/the-podfile.html)
-of an Xcode project. After that the framework built from the Kotlin/Native module can be used from
-this Xcode project. If necessary, this framework is automatically rebuilt during Xcode build process.
+1. Add a dependency on a Pod library stored locally with `pod()` to `build.gradle.kts` (`build.gradle`) of your
+ project.  
+As the third parameter, specify the path to `Podspec` of the local Pod using `project.file(..)`.  
```

Comment:
As the third parameter

where does third come from?

Reply:
from the example. project.file() is the third parameter after the name and version.
`pod("pod_dependency", "1.0", project.file("../pod_dependency/pod_dependency.podspec"))`

Reply:
Ahh perfect, thanks!